src/ui-kit/NavButton.jsx

```diff
@@ -12,15 +19,48 @@ const useStyles = makeStyles((theme) => ({
     fontSize: "14px",
   },
 }));
-const NavButton = (props) => {
-  const { pageName, url, theme } = props;
+const NavButton = ({ page, theme }) => {
+  const { name, url, subpages } = page;
   const classes = useStyles(theme);
 
-  return (
-    <Link className={classes.navLink} href={url}>
-      <Typography className={classes.navText}>{pageName}</Typography>
-    </Link>
-  );
+  const [anchorE1, setAnchorE1] = useState(null);
+
+  const handleClick = (event) => {
+    setAnchorE1(event.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorE1(null);
+  };
+  if (subpages) {
+    return (
+      <div>
+        <Button className={classes.navButtonStyle} onClick={handleClick}>
+          <Typography className={classes.navText}>{name}</Typography>
+          <ExpandMoreOutlined />
+        </Button>
+
+        <Menu
+          id={"subpages menu"}
+          anchorEl={anchorE1}
```

Comment:
What is this prop doing? Is this something we have control over naming? If so, could we name it to be something a bit more descriptive?

Reply:
it is the anchor for the menu popup, it is a existing prop from mui, i may be able to rename it, i am not sure.

Reply:
Ah, okay, I see. Anchor el makes sense now. Thanks!